### DISCUSSION TITLE
can't install anything through crouton for the last day, CRAS

### DISCUSSION THREAD

**Discussion by PieterBelska** [dnschneid/crouton#4788] [General]
can't install anything through crouton for the last day, CRAS

**Comment by iptoux** [comment]
> can't install anything through crouton for the last day, CRAS

Did you run in error: #4804?

There is an fix available 👍

**Comment by PieterBelska** [comment]
Thanks so much

**Reply by iptoux** [reply]
Please mark/edit issue as fixed 👍

**Comment by TomTravis** [comment]
did https://github.com/dnschneid/crouton accept the changes and apply  to the current  files
or submit a pull request
to update the image
https://goo.gl/fd3zc
